src/task/metrics/mean_rowwise_error/script.py

```diff
@@ -20,19 +21,23 @@
 
 print("Calculate mean rowwise RMSE", flush=True)
 mean_rowwise_rmse = 0
+mean_rowwise_mae = 0
 for i in de_test.index:
-    mean_rowwise_rmse += ((de_test.iloc[i] - prediction.iloc[i])**2).mean()
+    diff = de_test.iloc[i] - prediction.iloc[i]
+    mean_rowwise_rmse += np.sqrt((diff**2).mean())
+    mean_rowwise_mae += np.abs(diff).mean()
 
 mean_rowwise_rmse /= de_test.shape[0]
+mean_rowwise_mae /= de_test.shape[0]
```

Comment:
@szalata Just to make sure, could you review this? If it's okay, then you can merge.